### PR TITLE
Update links on README and references to Safe Multisig

### LINKED ISSUE #295: Broken environment links in README
## Description
The links to deployed environments in the README are broken:

> When the code is merged into development: https://safe-apps.dev.gnosisdev.com
> When the code is merged into master: https://safe-apps.staging.gnosisdev.com
> When the code is released: https://apps.gnosis-safe.io

## Expected result
Each link directs to the working deployment.

## Obtained result
All three are broken.

### PR DESCRIPTION
**Pull request by dasanra**
## What it solves
Resolves #295 

## How this PR fixes it
Update links and references in README file to be updated with reality. Also updates license and badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,11 @@
 # Gnosis Safe Apps 
 
-[![Logo](https://raw.githubusercontent.com/gnosis/safe-react-apps/main/assets/logo.png)](https://gnosis.pm/)
+[![Logo](https://raw.githubusercontent.com/gnosis/safe-react-apps/main/assets/logo.png)](https://gnosis-safe.io/)
 
-[![Build Status](https://travis-ci.com/gnosis/safe-react-apps.svg?branch=main)](https://travis-ci.com/gnosis/safe-react-apps)
+![license](https://img.shields.io/github/license/gnosis/safe-react-apps)
+![build](https://img.shields.io/github/workflow/status/gnosis/safe-react-apps/Deploy%20safe%20apps/main)
```
Comment (DaniSomoza) [inline README.md:7]: What is your opinion about to add an extra badges for tests:

```
![tests](https://img.shields.io/github/workflow/status/gnosis/safe-react-apps/Test/main?label=tests)
```

And In the future we could add this repo to coveralls (maybe when we finish the new designs in the Tx builder) and show the coverage badge :D

File: README.md [hunk 2]
```diff
@@ -63,15 +65,28 @@ headers: {
...
-* When the code is merged into master: https://safe-apps.staging.gnosisdev.com
-* When the code is released: https://apps.gnosis-safe.io
+These apps are deployed in the following environments.
+* When the code is merged into `development`: https://safe-apps.dev.gnosisdev.com
 
-## License
+    [Compound](https://safe-apps.dev.gnosisdev.com/compound)
+    | [Drain Account](https://safe-apps.dev.gnosisdev.com/drain-safe)
+    | [Transaction Builder](https://safe-apps.dev.gnosisdev.com/tx-builder)
+    | [WalletConnect](https://safe-apps.dev.gnosisdev.com/wallet-connect)
```
Comment (DaniSomoza) [inline README.md:74]: Should we add here the new ramp-network Safe app ?

File: README.md [hunk 3]
```diff
@@ -42,13 +44,13 @@ yarn start-wallet-connect
```
Comment (DaniSomoza) [inline README.md:44]: and also in line 34 add the commands are now:
```bash
yarn start:compound
yarn start:drain-safe
yarn start:tx-builder
yarn start:walletconnect
```
And also add the new ramp safe app script:
```
yarn start:ramp
```

Comment (dasanra) [reply README.md:44]: This is already in place, but we are missing the merge from development. Will do

File: README.md (hunk 1 above)
Comment (mikheevm) [reply README.md:7]: Do we have tests?

**Review by mikheevm — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (dasanra) [reply README.md:8]: Yes, at least Drain Safe and TX-Builder have some tests

Comment (DaniSomoza) [reply README.md:8]: and we have tests in the Walletconnect Safe App! :D 
https://github.com/gnosis/safe-react-apps/blob/development/apps/wallet-connect/src/App.test.tsx

**Pull request merged**